README.md

```diff
@@ -262,6 +262,28 @@ spec:
     restartPolicy: ""
 ```
 
+## Routing reference
+
+Below you will find a list of all accepted input parameters to the `Routing`
+custom resource. Only types are shown here. The fields are documented in the API, see [the API](https://pkg.go.dev/github.com/kartverket/skiperator@v1.0.0/api/v1alpha1)
+```yaml
+apiVersion: skiperator.kartverket.no/v1alpha1
+kind: Routing
+metadata:
+  name: sample-routing
+  namespace: sample
+spec:
+  hostname: app.example.com
+  redirectToHTTPS: true
```

Comment:
Consider not including this line in the example as no sane person would want to expose plain HTTP in 2024